README.md

```diff
@@ -78,21 +77,22 @@ Now that you have more data for each autocomplete item, you'll want to customize
 The default rendering function is shown below:
 
 	var defaultRenderFunction = function(document_type, item) {
```

Comment:
Interestingly, the `defaultRenderFunction` listed is incorrect. It's actually:

```
var defaultRenderFunction = function(document_type, item) {
    return '<p class="title">' + Swiftype.htmlEscape(item['title']) + '</p>';
  };
```

So, that's another way to sanitize your data. Anywhere you're doing `item.highlight[]`, you COULD do `Swiftype.htmlEscape(item[])`. Whichever the preferred solution is.